colossalai/zero/sharded_optim/low_level_optim.py

```diff
@@ -225,17 +235,21 @@ def _initialize_optimizer_states(self):
             fp32_partition_grad = torch.zeros_like(fp32_partition_param)
             fp32_partition_param.grad = fp32_partition_grad
 
+        # we do not need log information for optimizer, so comment them
         # update the parameter with zero gradients for initialization of optimizer states
-        self._optimizer.step()
+        # self._optimizer.step()
```

Comment:
This step is not for logging, but for optimizer state initialization.

Reply:
it influences the step of Adam.

Reply:
In what way？